doc/subsystems/test/sanitycheck.rst

```diff
@@ -3,20 +3,19 @@ Zephyr Sanity Tests
 ###################
 
 This script scans for the set of unit test applications in the git repository
-and attempts to execute them. By default, it tries to build each test case
-boards set to be default in the board definition file.
```

Comment:
it should be: "By default, it tries to build each test case on boards marked as default in the board definition file"

Reply:
that makes more sense, thanks.